src/pages/seulbiKim/Login/LoginSb.js

```diff
@@ -1,7 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './LoginSb.scss';
 
-function Login() {
-  return <div>Login</div>;
-}
+const LoginSb = () => {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/main-seulbiKim');
+  };
 
-export default Login;
+  let [input, setInput] = useState({
+    id: '',
+    pw: '',
+  });
+  const { id, pw } = input;
+  const isInputValid = id.includes('@') && pw.length >= 5 ? false : true;
```

Comment:
```suggestion
  const isInputValid = id.includes('@') && pw.length >= 5;
```
삼항연산자를 쓰지 않아도 조건에 일치하면 isInputValid는 true를 갖게 됩니다.

Reply:
넵! 수정하면서 button 속성 값도 변경했습니다